core/templates/dev/head/filters.js

```diff
@@ -240,6 +240,7 @@ oppia.filter('parameterizeRuleDescription', [
           replacementText = inputs[varName];
         }
 
+        replacementText = replacementText.replace('$', '$$$$');
```

Comment:
Hi @nithusha21, `TypeError: n.replace is not a functiontion`(from travis log)
 PTAL!

Reply:
Thanks! Apparently replacementText wasn't a string always... I was carried away by the name😅